__tests__/integration.test.js

```diff
@@ -66,6 +66,7 @@ describe("GET /api/articles/:article_id", () => {
           expect(typeof property.created_at).toBe("string");
           expect(typeof property.votes).toBe("number");
           expect(typeof property.article_img_url).toBe("string");
+          expect(property.comment_count).toBe("11");
```

Comment:
Ideally you dont change old tests. 

You've added a new behaviour, so following TDD, would make a new test to check for this. 
It would be something that essentially checks for this key and value - you're previous tests will check for the other keys etc.